spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebMVCSecurity.java

```diff
@@ -27,12 +27,12 @@ public InMemoryUserDetailsManager userDetailsService() {
 
     @Bean
     public WebSecurityCustomizer webSecurityCustomizer() {
-        return (web) -> web.ignoring().antMatchers("/resources/**");
+        return (web) -> web.ignoring().requestMatchers("/resources/**");
```

Comment:
I guess the parentheses around web can be removed.

Reply:
correct, I have done the changes